_posts/2016-09-22-foreman-community-newsletter---september-2016.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Foreman Community Newsletter - September 2016
+date: 2016-09-25 10:57:26
+author: Greg Sutcliffe
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Foreman 1.13 highlights, Katello 3.1 and 3.2-RC1, EL6 deprecated for 1.14 and all the
+  usual plugin and media roundup
+---
+
+### Foreman 1.13 RC1 released
+
+Foreman 1.13 RC1 (and RC2 very soon!) is now available for general testing -
+and this is a big one!  We've got the initial work on IPv6 (addressing and
+partial orchestration), UEFI bootloader support, improvements to DHCP subnet
+parsing, Compute Resource upgrades (notably OpenStack Liberty), and support for
+Facter 2's structured facts - and that's just the headlines!
+
+With so many new features, your help is needed more than ever to test this
+release and ensure it meets the rock-solid quality expected of our releases.
+You can find out more and get involved from the [announcement][1_13_rc1] and
+please do report any [bugs][bugs]. Thanks!
+
+[1_13_rc1]: https://groups.google.com/d/topic/foreman-announce/klyCElDqal0/discussion
+[bugs]: http://projects.theforeman.org/projects/foreman/issues/new
+
+### 1.12.3 released
+
+The latest point release of the Foreman 1.12 series is now available, bringing
+bugfixes and support for Puppet Server 2.6. You can read all the details in the
+[announcement][1_12_3]
+
+[1_12_3]: https://groups.google.com/d/topic/foreman-users/0njkhkpRWzM/discussion
+
+### EL6 deprecated in 1.13, to be removed in 1.14
+
+Users of EL6 / Centos6 should be aware that 1.13 will be the last release with
```

Comment:
Add a disclaimer that RC packages are not available in the repos yet, people might look for them